### ISSUE TITLE
Add Accessibility Attributes to btn: and menu: Macros

### ISSUE THREAD

**Issue by jcayouette** [asciidoctor/asciidoctor#4688]
# Extend macro syntax to support alt, aria-label, or title attributes

Both btn: and menu: macros currently:

Render stylized text or symbols (sometimes as `<kbd>` or `<span>`), with no aria-label, alt, or title.

Fail accessibility audits (e.g., Siteimprove) due to lack of screen reader support.

## Button and Menu Macros

Example Extended Syntax (Proposed):
```Adoc
btn:[Save, role="button",title="Save changes", alt="Save changes"]

menu:File[Open > Recent > Last Used, title="Open the last used file", aria-label="File menu, Open submenu, Recent, Last Used"]
```

**Comment by mojavelinux** [comment]
These macros are really just meant to be stylized text. I don't see the need for title and alt since the text is already self explanatory. This just doesn't seem like an important change to me. Do we really improve the reading experience in any tangible way?
